pom.xml

```diff
@@ -1,470 +1,511 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
```

Comment:
why do you want to change the default tabulation configuration of IntelliJ and Visual Studio Code (tabulation = 4 characters) ?
For me, your modification is a personal modification and I think we have to keep standard IntelliJ / Visual Studio Code format rules to be homogeneous.